curvefs/src/client/fuse_client.cpp

```diff
@@ -122,17 +122,241 @@ CURVEFS_ERROR FuseClient::Init(const FuseClientOption &option) {
     if (ret3 != CURVEFS_ERROR::OK) {
         return ret3;
     }
-
     ret3 =
         dentryManager_->Init(option.dCacheLruSize, option.enableDCacheMetrics);
     if (ret3 != CURVEFS_ERROR::OK) {
         return ret3;
     }
-
+    warmUpFile_.exist = false;
+    bgCmdStop_.store(false, std::memory_order_release);
+    bgCmdTaskThread_ = Thread(&FuseClient::BackGroundCmdTask, this);
+    taskFetchMetaPool_.Start(WARMUP_THREADS);
     return ret3;
 }
 
+void FuseClient::BackGroundCmdTask() {
```

Comment:
Why is warmup placed here? Does the volume backend also need warmup?

Reply:
yes， volume backend also need warmup，we will do it after we complete the uniform cache layer